src/onegov/core/orm/session_manager.py

```diff
@@ -683,6 +683,8 @@ def ensure_schema_exists(self, schema: str) -> None:
                 for base in self.bases:
                     base.metadata.schema = None
 
+                conn.execute('COMMIT')
+
```

Comment:
Explain why you are doing this with a comment. This looks fragile. Are you working around exceptions here to ensure the commit always happens?

Maybe we should switch to using conn as a context manager i.e. `with conn:` and get rid of the manual commit.

Reply:
This line came in while fixing tests. The line is not needed.